### PR TITLE
Update supplementalData.txt and units.txt from upstream #1173

### PR DESCRIPTION
**Pull request by hugovdm**
This grabs the output from an LdmlConverter run from this branch:
https://github.com/unicode-org/icu/pull/1173
(Once that branch is submitted, our resource file should automatically stay fresh when new CLDR data is imported.)

I'll take care of merging this into PR #30 if this PR gets submitted before that one. I tested already: post-merge, the unit tests all still pass, and units.txt diff should disappear from PR #30.

##### Checklist

- [ ] Issue filed: https://unicode-org.atlassian.net/browse/ICU-_____
- [ ] Updated PR title and link in previous line to include Issue number
- [ ] Issue accepted
- [ ] Tests included
- [ ] Documentation is changed or added

### CODE REVIEW

File: tools/cldr/cldr-to-icu/src/main/java/org/unicode/icu/tool/cldrtoicu/LdmlConverter.java
Comment (hugovdm) [inline tools/cldr/cldr-to-icu/src/main/java/org/unicode/icu/tool/cldrtoicu/LdmlConverter.java]: Nope. This is painful / won't work - since we're stale w.r.t. upstream/master. I'm removing this file from this PR.

**Review by sffc — APPROVED** [review]

**Comment by sffc** [comment]
Actually, wait, Travis is failing. Can you check why before merging this? I really want to keep Travis passing on units-staging.

**Review by younies — APPROVED** [review]

**Comment by hugovdm** [comment]
> Actually, wait, Travis is failing. Can you check why before merging this? I really want to keep Travis passing on units-staging.

Absolutely agreed! And <sigh> - I'm considering workflow or discipline improvements to catch this before sending reviews.

I lacked the unit-test adjustment needed by the inclusion of the "inch" preference. (I cherry-picked it from units_router branch and gave it a better commit message.)

**Comment by hugovdm** [comment]
This is units-staging: I won't squash; I will merge when the checks show they are all happy. Thanks for the approvals.

**Pull request merged**
